### ISSUE TITLE
 pretrain model's rendered results look strange

### ISSUE THREAD

**Issue by AmazingRoad** [OPEN-AIR-SUN/mars#72]
Hi, the following result is I recontruct with the pretrain model, and it looks strage. The foreground car is blur, even in the background. Could you share the wandb report of your pre-train model?

![image](https://github.com/OPEN-AIR-SUN/mars/assets/53403532/949d1a5c-d07e-4342-9c27-57ad169cd381)

**Comment by xBeho1der** [comment]
Here is an [wandb log](https://api.wandb.ai/links/wuzirui-research/w7ob6zt7) of our experiment on KITTI 0006. Notice that this experiment was not carried out in the release version.

**Comment by AmazingRoad** [comment]
> wandb log

Thanks! 
Is this is a reconstruction task? I find the pnsr in this log is different from the repo which is about 29.
And I train with the default code, the pnsr is about 23.

**Comment by xBeho1der** [comment]
Hi! This link is a reconstruction task. During this experiment, there was a bug related to evaluating all images that had not been fixed yet. We have chosen the `Eval Images Metrics/PSNR` as the evaluation metric.

**Comment by AmazingRoad** [comment]
> Hi! This link is a reconstruction task. During this experiment, there was a bug related to evaluating all images that had not been fixed yet. We have chosen the `Eval Images Metrics/PSNR` as the evaluation metric.

Hi, another question, If the depth loss in your log is normal? It is a little large, which I doubt if it is another. bug.
![image](https://github.com/OPEN-AIR-SUN/mars/assets/53403532/5fe6b88b-a6a4-472b-854a-5ef938559d72)

**Comment by wuzirui** [comment]
Hmmmm the high depth loss is because of the gt depth mask (depth completion used here, that's why we turned to monocular), and there might be some problem in our checkpoints, I'll see if we can find one (please bear with us because they are so many, and the wandb logs break from time to time...) or train another one (but this would only be happened after the refactor...).

As to the bad performance in your first image, could you provide the rendering of other frames (when the ego car haven't started to move yet)? I am thinking whether is the problem with first/last frame. Thanks!

**Comment by AmazingRoad** [comment]
> depth completion

@wuzirui As the following image, it seems look fine if the ego-car is static.  My own model trained with kitti alos has the same problem(the background becomes blur when ego-car begins to move)
![image](https://github.com/OPEN-AIR-SUN/mars/assets/53403532/2041ca01-6ade-4aed-8c2f-49e7d13f12bc)

**Comment by wuzirui** [comment]
just from this image, the scale factor indeed has problems. We will work on that, thank you for the feedback.

**Comment by AmazingRoad** [comment]
> just from this image, the scale factor indeed has problems. We will work on that, thank you for the feedback.

Thanks! I will also try some other parameters.

**Comment by wuzirui** [comment]
Hi @AmazingRoad , I am closing this issue for inactivity, but feel free to reopen :)
